bin/tee-key-preexec/src/main.rs

```diff
@@ -55,14 +59,25 @@ fn main_with_error() -> Result<()> {
         }
     };
 
-    let err = Command::new(&args[1])
-        .args(&args[2..])
-        .env("TEE_SIGNING_KEY", signing_key.display_secret().to_string())
-        .env("TEE_QUOTE_FILE", TEE_QUOTE_FILE)
-        .env("TEE_TYPE", tee_type)
+    let err = Command::new(&args.cmd_args[0])
+        .args(&args.cmd_args[1..])
+        .env(
+            format!("{}SIGNING_KEY", args.env_prefix),
+            signing_key.display_secret().to_string(),
```

Comment:
`.to_string()` [seems](https://docs.rs/secp256k1/latest/secp256k1/struct.SecretKey.html#method.display_secret) to be redundant here:
```suggestion
            signing_key.display_secret(),
```

Reply:
```
66  |             signing_key.display_secret(),
    |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^ the trait `std::convert::AsRef<std::ffi::OsStr>` is not implemented for `secp256k1::secret::DisplaySecret`
    |
```

Reply:
Supposedly doesn't work. Ignore.